solidity/contracts/bank/Bank.sol

```diff
@@ -22,16 +22,16 @@ import "../vault/IVault.sol";
 
 /// @title Bitcoin Bank
 /// @notice Bank is a central component tracking Bitcoin balances. Balances can
-///         be transferred between holders and holders can approve their
+///         be transferred between holders, and holders can approve their
 ///         balances to be spent by others. Balances in the Bank are updated for
-///         depositors who deposit their Bitcoin into the Bridge and only the
+///         depositors who deposited their Bitcoin into the Bridge and only the
 ///         Bridge can increase balances.
 /// @dev Bank is a governable contract and the Governance can upgrade the Bridge
 ///      address.
 contract Bank is Ownable {
     address public bridge;
 
-    /// @notice The balance of a given account in the Bank. Zero by default.
+    /// @notice The balance of the given account in the Bank. Zero by default.
     mapping(address => uint256) public balanceOf;
 
     /// @notice The remaining amount of balance a spender will be
```

Comment:
In the docstring for `Bank`, we say that balances are transferred between "holders", and in `allowance`, we say that a "spender" can transfer on behalf of an "owner".

When is an "owner" different than a "holder"?